extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -39,4 +75,140 @@ function renderVideoAccordian(document, videoObjects) {
+function renderActiveFeedback(video, response) {
+    let color= "#5959e6";
+    let activeChart = document.getElementById(`activeFeedback_${video.videoID}`).getContext("2d");
+    response.active_questions.sort((a, b) => {
+        return a.timestamp - b.timestamp;
+    });
+    let labels = response.active_questions.flatMap((question) => {
+        if (question.timestamp !== null) {
+            return [question.timestamp];
+        } else {
+            return [];
+        }
+    });
+    let amounts = response.active_questions.flatMap(question => {
+        if (question.count != null) {
+            return [question.count];
+        } else {
+            return [];
+        }
+    });
+    let questionsMap = new Map();
+    labels.forEach((timestamp) => {
+        questionsMap[timestamp] = response.active_questions_text.filter(question => question.timestamp === timestamp);
+    });
+    let chart = new Chart(activeChart, {
+        type: "bar",
+        data: {
+            labels: labels,
+            datasets: [{
+                label: "Active Feedback",
+                borderColor: color,
+                pointBackgroundColor: color,
+                pointBorderColor: color,
+                pointHoverBackgroundColor:color,
+                pointHoverBorderColor: color,
+                data: amounts,
+            }],
+        },
+        options: {
+            scales: {
+                xAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: "Timestamp (s)"
+                    },
+                    ticks: {
+                        beginAtZero: true
+                    }
+                }], 
+                yAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: "# of Questions"
+                    },
+                    ticks: {
+                        beginAtZero: true
+                    }
+                }]
+            }
+        }
+    });
+    let canvas = document.getElementById(`activeFeedback_${video.videoID}`);
+    canvas.onclick = function(evt) {
+        let firstPoint = chart.getElementAtEvent(evt)[0];
+
+        if (firstPoint) {
+            let label = chart.data.labels[firstPoint._index];
+            console.debug(label, questionsMap[label]);
+            displayStudentQuestions(label, questionsMap);
+        }
+    };
+}
+
+function renderPassiveFeedback(video, response) {
+    let color= "#5959e6";
+
+    let passiveChart = document.getElementById(`passiveFeedback_${video.videoID}`).getContext("2d");
+    response.passive_question.sort((a, b) => {
+        return a.timestamp - b.timestamp;
+    });
+    let transformedData = response.passive_question.map((question) => { 
+        return { x: question.timestamp, y: question.count };
+    });
+    let labels = response.passive_question.flatMap((question) => {
+        if (question.timestamp !== null) {
```

Comment:
same as above

Reply:
0 is  not truthy unfortunately.
if (question.timestamp) is what I used initially.